src/com/tictacgo/data/Piece.java

```diff
@@ -63,7 +63,7 @@ public Piece(int posx, int posy, int dirx, int diry, Player player, int sideLeng
 		direction[1] = diry;
         this.player = player;
 		this.sideLength = sideLength;
-		updateImageResource(); // Initialize which drawable to use for the Piece
+		updateImageResource(true); // Initialize which drawable to use for the Piece
```

Comment:
Either comment what true means, e.g. "updateImageResource(true /*fullPiece*/)", or break updateImageResource into two methods to cover the two cases (I'd prefer the latter).